grunt/watch.js

```diff
@@ -1,60 +1,36 @@
+"use strict";
+
 module.exports = {
-    options:
-    {
-        atBegin: true,
```

Comment:
Because `watch` was moved to the `npm start` task, which runs a build prior to running watch, making it redundant